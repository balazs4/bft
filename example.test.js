const { test } = require('./');
const assert = require('assert').strict;

test(`it should pass async`, async () => {
  await new Promise((resolve) => setTimeout(resolve, 500));
  assert.strictEqual(2, 2);
});

test(`it should pass`, () => {
  assert.strictEqual(2, 2);
});

test(`it should fail`, () => {
  assert.strictEqual('foobar', 'foobarrrrrrrr');
});

test(`it should skip`);

test(`it should fail async`, async () => {
  await new Promise((resolve) => setTimeout(resolve, 250));
  assert.strictEqual(3, 2);
});
