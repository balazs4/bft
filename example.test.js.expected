FAILED	it should fail
	Expected values to be strictly equal:
	[32m+ actual[39m [31m- expected[39m
	
	[32m+[39m 'foobar'
	[31m-[39m 'foobarrrrrrrr'
	         ^
PASSED	it should pass
SKIPPED	it should skip
PASSED	it should pass
